components/espcoredump/include_core_dump/esp_core_dump_types.h

```diff
@@ -146,6 +146,8 @@ typedef struct _core_dump_header_t {
     uint32_t tcb_sz;    /*!< Size of a TCB, in bytes */
     uint32_t mem_segs_num; /*!< Number of memory segments */
     uint32_t chip_rev; /*!< Chip revision */
+    uint64_t boot_time; /*!< Boot time, in microseconds */
+    uint64_t unix_time; /*!< Unix time, in microseconds */
```

Comment:
To avoid changing the header format (which requires changing the COREDUMP_VERSION...), could we keep this information in an ELF info segment? That way, the change would not cause compatibility issues even if an older version of esp-coredump package is used to decode the new core dump binary.

(Of course, this feature would be specific to the ELF format, but BIN format is present for backwards compatibility only, so not adding the feature there is okay.)